sensor/ACLAuditLogPlugin.js

```diff
@@ -288,6 +308,47 @@ class ACLAuditLogPlugin extends Sensor {
     this.writeBuffer(mac, record)
   }
 
+  // line example
+  // [Blocked]ts=1620435648 mac=68:54:5a:68:e4:30 sh=192.168.154.168 sh6= dn=hometwn-device-api.coro.net
+  // [Blocked]ts=1620435648 mac=68:54:5a:68:e4:30 sh= sh6=2001::1234:0:0:567:ff dn=hometwn-device-api.coro.net
+  async _processDnsmasqLog(line) {
+    if (line && 
+      line.includes("[Blocked]") &&
+      !line.includes("FF:FF:FF:FF:FF:FF")) {
+      const recordArr = line.substr(line.indexOf("[Blocked]") + 9).split(' ');
+      const record = {};
+      record.rc = 3; // dns block's return code is 3
```

Comment:
we need to change the condition of drop / accept check:
1. queries from dnsmasq log => always drop
2. queries from bro log => accept if rc != 3